fundamentals/middleware/rate-limit/WebRate2/Controllers/Home2Controller.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.RateLimiting;
+using System.Diagnostics;
+using WebMVCrm.Models;
+
+namespace WebMVCrm.Controllers;
+// <snippet_1>
+[EnableRateLimitingAttribute("fixed")]
+public class Home2Controller : Controller
+{
+    private readonly ILogger<Home2Controller> _logger;
+
+    public Home2Controller(ILogger<Home2Controller> logger)
+    {
+        _logger = logger;
+    }
+
+    public ActionResult Index()
+    {
+        return View();
+    }
+
+    [EnableRateLimitingAttribute("sliding")]
```

Comment:
`[EnableRateLimitingAttribute("sliding")]` overrides controller limit `[EnableRateLimitingAttribute("fixed")]` with the `#define SECOND` in Program.cs where I have
`app.MapDefaultControllerRoute();`
But in Program.cs with `#define FIRST` I have
`app.MapDefaultControllerRoute().RequireRateLimiting(fixedPolicy);`

In that case, `[DisableRateLimiting]` works, but `[EnableRateLimitingAttribute("sliding")]` isn't applied and the sliding policy is used.